organizing-containers-of-balls.ts

```diff
@@ -0,0 +1,16 @@
+export function organizingContainers(container: number[][]): string {
+  if (container.length === 1 && container[0].length > 1) return 'Impossible'
+
+  const row = new Array(container.length).fill(0)
+  const column = new Array(container.length).fill(0)
```

Comment:
does it mean the number of containers and balls are always the same?

Reply:
The first line contains an integer n, the number of containers (rows) and ball types (columns). This is from hackerrank